### PR TITLE
Update getting-started guide KVM and CPU commands

### PR DESCRIPTION
**Pull request by xavinux**
Commands on Linux to see if virtualization is supported and KVM is enabled.

Signed-off-by: Javier Romero <xavinux@gmail.com>

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -17,22 +17,46 @@ instance using Ubuntu 18.04 on EC2. Firecracker uses
...
+  
+  ```
+  egrep '(vmx|svm)' --color=always /proc/cpuinfo
+  ```
+  you should see the following answer (number of flags depends of the number of cores on the CPU):
+  
+  ```
+  flags		: fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ht syscall nx   mmxext fxsr_opt pdpe1gb rdtscp lm 3dnowext 3dnow constant_tsc rep_good nopl nonstop_tsc cpuid extd_apicid pni monitor cx16   popcnt lahf_lm cmp_legacy svm extapic cr8_legacy abm sse4a misalignsse 3dnowprefetch osvw ibs skinit wdt nodeid_msr         hw_pstate vmmcall npt lbrv svm_lock nrip_save
+  flags		: fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ht syscall nx   mmxext fxsr_opt pdpe1gb rdtscp lm 3dnowext 3dnow constant_tsc rep_good nopl nonstop_tsc cpuid extd_apicid pni monitor cx16   popcnt lahf_lm cmp_legacy svm extapic cr8_legacy abm sse4a misalignsse 3dnowprefetch osvw ibs skinit wdt nodeid_msr         hw_pstate vmmcall npt lbrv svm_lock nrip_save
+
```
Comment (iggy) [inline docs/getting-started.md]: It might be nice to show an example from Intel and ARM CPUs.

Comment (xavinux) [reply docs/getting-started.md]: For ARM architecture I have a Raspberry PI 3, will see how KVM can be enabled on it's Linux Kernel an if everything works fine, will post also an example with ARM CPU.

File: docs/getting-started.md [hunk 2]
```diff
@@ -17,22 +17,46 @@ instance using Ubuntu 18.04 on EC2. Firecracker uses
 [KVM](https://www.linux-kvm.org) and needs read/write access that can be
 granted as shown below:
 
-```
-sudo setfacl -m u:${USER}:rw /dev/kvm
-```
+ ```
```
Comment (aghecenco) [inline docs/getting-started.md]: Remove leading spaces for this line and the next 2.

File: docs/getting-started.md [hunk 3]
```diff
@@ -17,22 +17,46 @@ instance using Ubuntu 18.04 on EC2. Firecracker uses
...
 
   Firecracker currently supports physical Linux x86_64 hosts, running kernel
-  version 4.14 or later.
+  version 4.14 or later. Please check first if your Linux host supports vitualization
+  running as root the following command:
+  
+  ```
+  egrep '(vmx|svm)' --color=always /proc/cpuinfo
+  ```
+  you should see the following answer (number of flags depends of the number of cores on the CPU):
```
Comment (aghecenco) [inline docs/getting-started.md]: Add blank lines before (already present) and after (missing) code snippets.

File: docs/getting-started.md [hunk 4]
```diff
@@ -17,22 +17,46 @@ instance using Ubuntu 18.04 on EC2. Firecracker uses
...
+ sudo setfacl -m u:${USER}:rw /dev/kvm
+ ```
 
 The generic requirements are explained below:
 
 - **Linux 4.14+**
 
   Firecracker currently supports physical Linux x86_64 hosts, running kernel
-  version 4.14 or later.
+  version 4.14 or later. Please check first if your Linux host supports vitualization
```
Comment (aghecenco) [inline docs/getting-started.md]:
```suggestion
  version 4.14 or later. Please check first if your Linux host supports virtualization
```

File: docs/getting-started.md (hunk 1 above)
Comment (aghecenco) [reply docs/getting-started.md]: `Firecracker currently supports physical Linux x86_64 hosts` is mentioned a few lines above; I'd rather hold off ARM instructions in the getting started guide until ARM support is fully in.

File: docs/getting-started.md [hunk 5]
```diff
@@ -17,22 +17,46 @@ instance using Ubuntu 18.04 on EC2. Firecracker uses
...
-  Please make sure that:
-  1. you have KVM enabled in your Linux kernel, and
-  2. you have read/write access to `/dev/kvm`.
+  Please make sure that you have KVM enabled in your Linux kernel, to do so
+  you can run:
+  
+  ```
+  lsmod | grep '^kvm'
+  ```
+  you should see an answer like the following:
```
Comment (aghecenco) [inline docs/getting-started.md]: Add newline after the code snippet

File: docs/getting-started.md [hunk 6]
```diff
@@ -17,22 +17,46 @@ instance using Ubuntu 18.04 on EC2. Firecracker uses
...
+  
+  ```
+  kvm-intel             86016  0
+  kvm                   598016  1 kvm-intel
+  ```   
+  
+  You also need to have read/write access to `/dev/kvm`.
```
Comment (aghecenco) [inline docs/getting-started.md]: I'm curious as to why there are leading whitespaces on some lines and not on the others - until we figure that out, please adhere to the existing style and in this case add some leading spaces to align with the lines below.

Comment (xavinux) [reply docs/getting-started.md]: @aghecenco thank you for your suggestions. All corrections done.

**Pull request closed without merging**